src/test/resources/features/State.feature

```diff
@@ -0,0 +1,98 @@
+@State @smoke
+Feature: Get location information of a city using country and state
+
+  Scenario Outline: Get a successful response when sending a valid country, state and a city
+    Given The user has access to the zippopotamus endpoint
+    When The user performs a GET request with a <country>, a <state> and a <city>
+    Then The response status code should be 200
+    And The response body should contains <state> as state abbreviation
+
+    Examples:
+      | country | state      |city       |
+      | "us"    | "MA"       |"belmont"  |
+
+  Scenario Outline: Get a unsuccessful response when sending a invalid country, a state and a city
+    Given The user has access to the zippopotamus endpoint
+    When The user performs a GET request with a <invalidCountry>, a <state> and a <city>
+    Then The response status code should be 404
+
+    Examples:
+      | invalidCountry | state      |city       |
+      | "us "          | "MA"       |"belmont"  |
+      | " us"          | "MA"       |"belmont"  |
+      | " uus "        | "MA"       |"belmont"  |
+      | "uss"          | "MA"       |"belmont"  |
+
+  Scenario Outline: Get a unsuccessful response when sending a country, an invalid state and a city
+    Given The user has access to the zippopotamus endpoint
+    When The user performs a GET request with a <country>, a <invalidState> and a <city>
+    Then The response status code should be 404
+
+    Examples:
+      | country        | invalidState    |city       |
```

Comment:
We should delete the examples that contain spaces, since they break the URL and there is no possibility that the operation work.

Reply:
Done